sqlite/src/analyze.c

```diff
@@ -226,47 +228,40 @@ static void openStatTable(
   ** if they do already exist.
   */
 #if defined(SQLITE_BUILDING_FOR_COMDB2)
-  if( db->isExpert==0 ){
```

Comment:
My research suggests that attempting to CREATE the stat tables on-the-fly cannot currently work right, regardless of whether "expert" mode is enabled.  Perhaps this limitation should be fixed in the future?